src/main/pg/pg.h

```diff
@@ -114,10 +114,14 @@ extern const uint8_t __pg_resetdata_end[];
     struct _dummy                                                       \
     /**/
 
+#ifndef SRAM2
```

Comment:
Should probably go into `target/config_post.h`.

Reply:
I removed this as being unrelated to this intent of this PR. It was to move the config information out of DTCM memory as that is a valuable resource, but this should really be done as part of a larger consideration of such usage.